addons/mail/static/src/core/web/discuss_patch.xml

```diff
@@ -6,6 +6,9 @@
         </xpath>
         <xpath expr="//*[@t-ref='root']" position="inside">
             <MessagingMenu t-if="ui.isSmall"/>
+            <div t-if="store.channels.status === 'fetching'" class="d-flex position-absolute flex-grow-1 align-items-center justify-content-center h-100 w-100 o-pointer-events-none">
+                <h4 class="text-muted opacity-50 o-visible-short-delay"><b><i class="o-mail-Discuss-mobileLoading fa fa-circle-o-notch fa-spin"/></b></h4>
+            </div>
```

Comment:
I think it is centered on the whole page rather than on the direct parent (thread view?) which feels a bit off.